common/test/acceptance/tests/lms/test_lms_dashboard.py

```diff
@@ -302,7 +302,7 @@ def test_near_future_course_date(self):
         self.course_fixture.configure_course()
 
         start_date = TEST_DATE_FORMAT.format(dt=course_start_date)
-        expected_course_date = "Starts - {start_date} GMT".format(start_date=start_date)
+        expected_course_date = "Starts - {start_date} UTC".format(start_date=start_date)
```

Comment:
If it truly doesn't matter we might want to just allow either to pass, but at this point that's a nit.